devtools/x/src/changed_since.rs

```diff
@@ -15,15 +15,17 @@ pub struct Args {
 }
 
 pub fn run(args: Args, xctx: XContext) -> Result<()> {
-    let affected_set = changed_since_impl(&xctx, &args.base)?;
```

Comment:
this I feel a little bad about since the git_cli type isn't published from the x-core crate, can't use it as a parameter.